mcp_server/main.py

```
@@ -14,7 +13,7 @@
 import sys
 import ast
 import re
-from typing import Dict, List, Any, Optional, Union
+from typing import Dict, List, Any, Optional
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)